05_syntax_lab/01.py

```diff
@@ -1,5 +1,15 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Syntax excersize #1
+"""
+bigNum = 0
+number = 0
+
+while number < 10:
+    print "Please enter any number"

```

Comment:
Bonus: There's an empty line between the print and the input. Can you have both in the same line?